Tasks/google-play-release/GooglePlay.ts

```diff
@@ -162,19 +193,19 @@ function run() {
  * @param {Object} apkFile The apk file from which to attempt name extraction
  * @return {string} packageName Name extracted from package. null if extraction failed
```

Comment:
I think this document line needs to be updated.  Is there an occasion where this will return 'null'?  It appears to just throw.

Perhaps this method should have its name changed from 'tryGetPackageName' to 'getPackageName'?